### ISSUE TITLE
Historical ZQ1 blocks should have their state root hash set to zero

### ISSUE THREAD

**Issue by JamesHinshelwood** [Zilliqa/zq2#2054]
We don't know the state for these blocks, so lets not pretend we do.

**Comment by JamesHinshelwood** [comment]
Also we shouldn't try to execute transactions against these 'null' states. The results are meaningless and waste our time.

**Comment by DrZoltanFazekas** [comment]
Agree, `eth_call` and `eth_estimateGas` shall also not try to execute code on the unknown state of old ZQ1 blocks

**Comment by 86667** [comment]
FYI this is complete I'm waiting for syncing issues to be fixed before merging
